src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -70,6 +72,7 @@ internal CurlHandler()
                 throw new HttpRequestException(SR.net_http_client_execution_error);
             }
             SetCurlMultiOptions();
+            PollFunction(_multiHandle);
```

Comment:
Oops.. and if I remove it, HTTPS doesn't work! Not sure what I am missing in my logic